clion/src/main/kotlin/org/rust/clion/profiler/dtrace/RsDTraceProfilerProcess.kt

```diff
@@ -86,3 +93,5 @@ class RsDTraceProfilerProcess private constructor(
         }
     }
 }
+
+private val BUILD_212: BuildNumber = BuildNumber.fromString("212")!!
```

Comment:
Let's add `BACKCOMPAT: 2021.1` comment here as well. Just not to forget to drop it